go/vt/topotools/rebuild_keyspace.go

```diff
@@ -30,14 +30,16 @@ import (
 )
 
 // RebuildKeyspace rebuilds the serving graph data while locking out other changes.
-func RebuildKeyspace(ctx context.Context, log logutil.Logger, ts *topo.Server, keyspace string, cells []string, allowPartial bool) (err error) {
+func RebuildKeyspace(ctx context.Context, _ logutil.Logger, ts *topo.Server, keyspace string, cells []string, allowPartial bool) (err error) {
+	// TODO: logutil.Logger is unused, clean up call sites.
+
 	ctx, unlock, lockErr := ts.LockKeyspace(ctx, keyspace, "RebuildKeyspace")
 	if lockErr != nil {
 		return lockErr
 	}
 	defer unlock(&err)
 
-	return RebuildKeyspaceLocked(ctx, log, ts, keyspace, cells, allowPartial)
```

Comment:
Better to keep this and change it to `_` in the signature of `RebuildKeyspaceLocked`. I suspect at some point in the past we did log from that func and now we don't. There are simply too many call sites for it to be worth cleaning them up.